File: 백준/Bronze/23825. SASA 모형을 만들어보자/SASA 모형을 만들어보자.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] str = sc.nextLine().split(" ");
        int s = Integer.parseInt(str[0]);
        int a = Integer.parseInt(str[1]);

        int cnt = 0;

        while(s >= 2 && a >= 2)
        {
            cnt++;
            s -= 2;
            a -= 2;
        }

        System.out.println(cnt);
    }
}

File: 백준/Bronze/23825. SASA 모형을 만들어보자/SASA 모형을 만들어보자.cs
string[] str = Console.ReadLine().Split();

int s = int.Parse(str[0]);
int a = int.Parse(str[1]);

int cnt = 0;

while(s >= 2 && a >= 2)
{
    cnt++;
    s -= 2;
    a -= 2;
}

Console.WriteLine(cnt);